eve/workers/end2end/scripts/build-latest-commits.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+set -exu
+
+BUILD_DIR="build"
+
+CLOUDSERVER_IMAGE="registry.scality.com/zenko-dev/cloudserver"
+CLOUDSERVER_TAG="latest"
+CLOUDSERVER_REPO="git@github.com:scality/cloudserver.git"
+CLOUDSERVER_BRANCH="development/8.2"
+
+BACKBEAT_IMAGE="registry.scality.com/zenko-dev/backbeat"
+BACKBEAT_TAG="latest"
+BACKBEAT_REPO="git@github.com:scality/backbeat.git"
+BACKBEAT_BRANCH="development/8.2"
+
+S3UTILS_IMAGE="registry.scality.com/zenko/s3utils"
+S3UTILS_TAG="latest"
+S3UTILS_REPO="git@github.com:scality/s3utils.git"
+S3UTILS_BRANCH="development/8.1"
+
+UI_IMAGE="registry.scality.com/sf-eng/zenko-ui"
+UI_TAG="latest"
+UI_REPO="git@github.com:scality/zenko-ui.git"
+UI_BRANCH="development/1.0"
+
+E2E_IMAGE="registry.scality.com/zenko-dev/zenko-e2e"
+E2E_TAG="latest"
+
+build_dir() {
+  mkdir -p  ${BUILD_DIR}
+}
+
+build_latest_cloudserver() {
+  git clone ${CLOUDSERVER_REPO} -b ${CLOUDSERVER_BRANCH} ${BUILD_DIR}/cloudserver
+  docker build -t ${CLOUDSERVER_IMAGE}:${CLOUDSERVER_TAG} ${BUILD_DIR}/cloudserver
+  kind load docker-image ${CLOUDSERVER_IMAGE}:${CLOUDSERVER_TAG}
+}
+
+build_latest_backbeat() {
+  git clone ${BACKBEAT_REPO} -b ${BACKBEAT_BRANCH} ${BUILD_DIR}/backbeat
+  docker build -t ${BACKBEAT_IMAGE}:${BACKBEAT_TAG} ${BUILD_DIR}/backbeat
+  kind load docker-image ${BACKBEAT_IMAGE}:${BACKBEAT_TAG}
+}
```

Comment:
I believe that applying this for cloudserver and backbeat is not required. It is already done by their own CI in the `post-merge` stage that they have in place. [you can check it per example here](https://github.com/scality/backbeat/blob/development/8.2/eve/main.yml#L135)

In short, when something get's merged in the `development/*` branches it triggers a new job that build and push an image into  `registry.scality.com/zenko/cloudserver:latest-8.2.1` (example for cloudserver), the same is valid for backbeat and cloudserver. (not yet available for s3utils but can be done quite easily)

I also prefer to keep the build/push of the image in their own project so that the project are able to "release" themselve and other repository like Zenko or Z-O can retrieve the package without having to build it.

Also I may understand that the 3 digits version can be an issue, in that case I can make a PR today that translate that also add a tag like `latest-8.2`.

Let me know what you think

Reply:
I had looked at those repos and was unable to find images built from `development/8.2`, so I went for the cloning and build.

Reply:
I think backbeat currently have a issue with dev/8.2 (just a matter of changing the version referred on the `package.json`)

And I will add a little CI pipeline to s3utils and zenko-ui if it's not the case as well. so that you don't have to build it yourself.

Let me know what you think.

Reply:
that sound fine.

Reply:
Created this [PR](https://github.com/scality/backbeat/pull/1055) on backbeat that allow us to have `latest-x.y` tags on our registry.
So you'll have registry.scality.com/zenko/backbeat:latest-8.2

I'll go ahead and do the same for the other components.